Reconstruction/PFA/Pandora/GaudiPandora/src/PandoraPFAlg.cpp

```diff
@@ -474,11 +482,10 @@ StatusCode PandoraPFAlg::updateMap()
                     std::vector<edm4hep::MCParticle> v_mc;
                     m_CollectionMaps->collectionMap_MC [v.first] = v_mc;
                     for(unsigned int i=0 ; i< po->size(); i++) m_CollectionMaps->collectionMap_MC [v.first].push_back(po->at(i));
-                    std::cout<<"saved col name="<<v.first<<std::endl;
-                }
-                else{
-                std::cout<<"don't find col name="<<v.first<<std::endl;
+                    if(m_debug) std::cout<<"saved col name="<<v.first<<std::endl;
```

Comment:
For the debug output, consider the message service in Gaudi.

Reply:
no problem.